<launch>
    <node name="pcd2tsdf" type="pcd2tsdf" pkg="loop_closure" output="screen" clear_params="true">
        <rosparam command="load" file="$(find loop_closure)/params/pcd_params.yaml"/>
    </node>
</launch>
